src/build.props

```diff
@@ -11,7 +11,7 @@
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
     
     <!-- VersionPrefix denotes the current Semantic Version for the SDK. Must be updated before every nuget drop. -->
-    <VersionPrefix>2.1.0</VersionPrefix>
+    <VersionPrefix>2.1.1</VersionPrefix>
```

Comment:
>VersionPrefix [](start = 5, length = 13)

If this now denotes the "Semantic Version for the SDK" it should be called `SdkSemanticVersion`.

Reply:
You are right, i named it similarly yesterday but that broke local build! Unfortunately there is a .net core dependency to call it "versionPrefix" :/ If we dont have this explicitly set, it is defaulted to 1.0.0. So, we added the comment instead.  (I should have added this note as well i.e. the name has a dependency!)

---
In reply to: [282603197](https://github.com/microsoft/sarif-sdk/pull/1488#discussion_r282603197) [](ancestors = 282603197)